openssl1_1_1.Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM python:3.10
+LABEL description="Deploy Mage on ECS"
+ARG PIP=pip3
+USER root
+
+# Download ODBC headers for pyodbc
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+RUN curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list
+RUN apt-get -y update
+RUN ACCEPT_EULA=Y apt-get -y install msodbcsql18
+RUN apt-get -y install unixodbc-dev
+
+# Install NFS dependencies, and pymssql dependencies
+RUN apt-get -y install nfs-common freetds-dev freetds-bin
+
+### Install openssl 1.1.1 instead of the default openssl 3.0.9
+# (Install compiling library Make)
+RUN apt-get -y install make 
+
+# (single command that will download latest binaries, extract them, cd into the directory, compile configuration and then install the files)  
+RUN wget https://www.openssl.org/source/openssl-1.1.1g.tar.gz
```

Comment:
Can it be installed based on base Mage docker image?

Reply:
hmm i could do that, but doesn't that mean we would have to wait until the base docker image is done building before creating this image?

Reply:
I would expect user to build the docker image themselves with this custom Dockerfile. It's similar with how we provide the Dockerfile for Mage with spark: https://github.com/mage-ai/mage-ai/tree/master/integrations/spark